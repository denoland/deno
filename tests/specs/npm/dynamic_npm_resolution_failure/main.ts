import chalk from "npm:chalk";

console.log(chalk.green("Hi"));
try {
  await import("npm:@denotest/dep-cannot-parse");
} catch (err) {
  console.log(err);
}
console.log(chalk.green("Bye"));
