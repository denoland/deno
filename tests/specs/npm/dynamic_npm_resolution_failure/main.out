[UNORDERED_START]
Download http://localhost:4260/chalk
Download http://localhost:4260/@denotest%2fdep-cannot-parse
[UNORDERED_END]
Download http://localhost:4260/chalk/chalk-5.0.1.tgz
Hi
TypeError: Error in @denotest/dep-cannot-parse@1.0.0 parsing version requirement for dependency "@denotest/esm-basic": "unknown-scheme:unknown"
    0: Invalid version requirement
    1: Unexpected character.
         unknown-scheme:unknown
         ~
    at async file:///[WILDLINE]main.ts:5:3 {
  code: "ERR_MODULE_NOT_FOUND"
}
Bye
